jobsapp/api/views/common.py

```diff
@@ -27,3 +34,31 @@ def get_queryset(self) -> "QuerySet[Job]":
             )
         else:
             return self.serializer_class.Meta.model.objects.filter(filled=False)
+
+
+@api_view(["POST"])
+@permission_classes([AllowAny])
+def contact_us(request: Request) -> Response:
+    serializer = ContactSerializer(data=request.data)
+    if serializer.is_valid():
+        contact_us_email(serializer.validated_data)
+        return Response(
+            {_("message"): _("Email sent successfully.")}, status=status.HTTP_202_ACCEPTED
+        )
+    return Response({_("message"): serializer.errors}, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["GET"])
+@permission_classes([AllowAny])
+def about_us(request: Request) -> Response:
+    # TODO: Get data from database
```

Comment:
We need to search if exists any dajngo's package for manage page's contents. 
We should create about's page content in admin.

Reply:
Something like django flat pages? https://docs.djangoproject.com/en/3.1/ref/contrib/flatpages/

Reply:
Yes, could be a good idea. Come on.